Dockerfile

```diff
@@ -40,20 +65,20 @@ RUN apt-get update && apt-get install --yes --no-install-recommends \
   && ln -sf /usr/bin/nodejs /usr/bin/node \
   && ln -sf /bin/bash /bin/sh
 
-ENV HOME=/home/captain \
-  WSBU_C_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-gcc \
-  WSBU_CXX_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-g++ \
-  WSBU_EMULATOR=/usr/bin/qemu-arm \
-  QEMU_LD_PREFIX=/opt/linaro/arm-linux-gnueabihf/libc \
-  CMAKE_TOOLCHAIN_FILE=/opt/toolchain-linaro-armhf.cmake
-
-COPY toolchain.cmake "$CMAKE_TOOLCHAIN_FILE"
-
+ENV GCC_PREFIX=/opt/linaro
 RUN git clone https://github.com/wsbu/linaro-release.git \
     --branch gcc-linaro-4.9.4-2017.01-x86_64_arm-linux-gnueabihf \
     --depth 1 \
-    /opt/linaro \
-  && rm -rf /opt/linaro/.git
+    "${GCC_PREFIX}" \
```

Comment:
It is used in the toolchain, so it's not a useless variable. I just prefer it being declared separately so that you can move the installation path of the compiler more easily. I don't have super strong feelings on declaring it separately, so I'd move it if you like, but I do think the benefits outweigh the slight decrease in legibility.

Reply:
Can you link to docs on toolchain ussage? If I google "GCC_PREFIX" I get conflicting usage. https://community.particle.io/t/feature-bounty-improve-our-makefile/567 uses it in place of CROSS_COMPILE, and http://phylo.bio.ku.edu/content/compiling-g-47 uses it like you are here, but usage seems sparse

Reply:
I'd still lean away from defining environment variables unless absolutely necessary. But that may be my preference.

Reply:
Okay, private chat yielded complete removal of the `GCC_PREFIX` variable and replaced with `sed`.